RELEASE_GUIDE.md

```diff
@@ -1,35 +1,63 @@
+- You need `spago` installed.
+- You need to be logged into NPM (i.e. running `npm whoami` should print your NPM account's username)
+
 ## Before making a release
 
 - Check that there are no unintended breaking changes by compiling [the latest package set](https://github.com/purescript/package-sets/releases/latest)
-- Check that INSTALL.md is up-to-date
-- Regenerate LICENSE: `make license-generator` (see `license-generator/` for
-  details)
-- Write release notes
 
-Additionally, if there are any breaking changes, there are number of downstream
-projects who we should probably at least notify:
+```bash
+stack build
+mkdir wPackageSet
+pushd wPackageSet
+spago init
+spago upgrade-set
+# install all packages in the set
+spago install $(spago ls packages | cut -f 1 -d ' ' | tr '\n' ' ')
+
+# Verify that code compiles and docs are properly created
+exec stack exec bash <<EOF
+spago build
+spago docs
+EOF
+popd
+# rm -rf wPackageSet
+```
+
+- Check that INSTALL.md is up-to-date. For example:
+    - Does the GHC version correspond to the one we're currently on?
+    - Is the list of supported OSes (based on GHC version) correct? If not, check the GHC website for the list.
+    - Are there any unofficial installation methods that should be dropped because they are no longer maintained?
+    - Are the instructions for building from source currently working?
+    - Anything else that needs to be changed?
+
+- Additionally, if there are any breaking changes, there are number of downstream
+projects whom we should probably at least notify. See below subsections:
 
 ### Libraries
 
 Are there breaking changes to the language? Or alternatively, are there
 language changes which require breaking changes in the relevant libraries to
-make use of? If so:
+make use of? If so, make a "meta" issue to track which libraries need to be changed and then update those libraries:
 
-- Update core libraries
-- Update contrib libraries
-- Update node bindings
-- Update web bindings
+- core libs: open a "meta" issue in [`purescript/purescript` issue](https://github.com/purescript/purescript/issues)). For example, [#3942](https://github.com/purescript/purescript/issues/3942)
+- contrib libs: open a "meta" issue in [`purescript-contrib/governance` issue](https://github.com/purescript-contrib/governance/issues). For example, [#35](https://github.com/purescript-contrib/governance/issues/35)
+- node libs: open a "meta" issue in [`purescript-contrib/governance` issue](https://github.com/purescript-contrib/governance/issues). For example, [#35](https://github.com/purescript-contrib/governance/issues/35)
+- web libs: open a "meta" issue in [`purescript-contrib/governance` issue](https://github.com/purescript-contrib/governance/issues). For example, [#35](https://github.com/purescript-contrib/governance/issues/35)
```

Comment:
Typo here? "Open a meta issue in purescript/purescript issue" doesn't make sense to me. I'm also not sure we need to repeat the whole "open a "meta" issue" spiel for every item in this list.

```suggestion
- core libraries
- contrib libraries
- node libraries
- web libraries

It may be helpful to open a "meta" issue in the relevant repository to track progress for each group of libraries which need updating. For example, [purescript/purescript#3942](https://github.com/purescript/purescript/issues/3942).
```